grammar.js

```diff
@@ -484,12 +485,14 @@ module.exports = grammar({
     catch_clause: $ => seq(
       keyword('catch'),
       '(',
-      field('type', $._type_name),
+      field('type', $.type_list),
       field('name', $.variable_name),
       ')',
       field('body', $.compound_statement)
     ),
 
+    type_list: $ => choice($._type_name, prec.right(pipeSep1($._type_name))),
```

Comment:
Could this just be written like this?

```js
type_list: $ => pipeSep1($._type_name),
```

Reply:
Good catch! That's almost a bit embarrasing 😄 Simplified now